src/main/java/uk/ac/cam/cl/dtg/isaac/dto/IsaacMultiChoiceQuestionDTO.java

```diff
@@ -27,9 +29,15 @@
 @JsonContentType("isaacMultiChoiceQuestion")
 public class IsaacMultiChoiceQuestionDTO extends IsaacQuestionBaseDTO {
 
-    @JsonIgnore
+    /**
+     * Gets the choices.
+     * 
+     * @return the choices
+     */
     @Override
-    public ContentBaseDTO getAnswer() {
-        return super.getAnswer();
+    @JsonIgnore(false)
+    public List<ChoiceDTO> getChoices() {
+        return super.choices;
```

Comment:
This is even worse than before! You need to use the getter, and not directly access the internal state.
```suggestion
        return super.getChoices();
```